Installation.md

```diff
@@ -54,6 +54,12 @@ Here's how to set up these packages:
 sudo apt-get update && sudo apt-get install git python-pip -y
 ```
 
+* On Ubuntu 16.04 need below additional packages:
+
+```
+sudo apt-get install python-cffi libffi-dev libssl-dev libcurl4-openssl-dev
```

Comment:
Originally the README didn't list those, since `./venv-dependencies.sh` would analyze your system and see if they're missing. There's no hurt in installing them up front though.